shell/platform/darwin/macos/framework/Source/FlutterConstants.h

```diff
@@ -0,0 +1,6 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// The name of the Info.plist flag to enable the embedded MacOS views preview.
+const char* const kEmbeddedViewsPreview = "io.flutter_embedded_views_preview";
```

Comment:
Don't need the constant in a separate file, can move it to `FlutterPlatformViewController.mm`